data/common.yaml

```diff
@@ -65,6 +65,9 @@ rabbitmq::ssl_reuse_sessions: true
 rabbitmq::ssl_honor_cipher_order: true
 rabbitmq::ssl_dhfile: ~
 rabbitmq::ssl_ciphers: []
+rabbitmq::ssl_crl_check: 'false'
```

Comment:
you don't need to add data to common.yaml since you already added them to the init.pp. only one of those two places is required (and init.pp is the preferred way)

Reply:
fixed